app/vmui/packages/vmui/src/pages/ExploreLogs/hooks/useFetchLogs.ts

```diff
@@ -50,12 +49,12 @@ export const useFetchLogs = (server: string, query: string) => {
         const lines = decoder.decode(value, { stream: true }).split("\n");
         result.push(...lines);
 
-        // Trim result to MAX_LINES
-        if (result.length > MAX_LINES) {
-          result.splice(0, result.length - MAX_LINES);
+        // Trim result to limit
+        if (result.length > limit) {
+          result.splice(0, result.length - limit);
```

Comment:
Should we trim the result if it will be limited from the backend?

Reply:
You're right, if there's a limit enforced on the backend, then this code is pointless. I think it's better to keep it for now, in case there is no limit on the backend, this code will ensure that the limit is not exceeded. I've added a comment, and in the future, this code can be removed.